test/barnard59.test.js

```diff
@@ -112,9 +112,9 @@ describe('barnard59', () => {
 
     it('should run the parse-csvw.ttl example without error', () => {
       const pipelineFile = (new URL('../examples/parse-csvw.ttl', import.meta.url)).pathname
-      const command = `${barnard59} run --pipeline=http://example.org/pipeline/parseCsvw ${pipelineFile}`
+      const command = `${barnard59} run -vv --pipeline=http://example.org/pipeline/parseCsvw ${pipelineFile}`
 
-      const result = shell.exec(command, { silent: true })
+      const result = shell.exec(command)
```

Comment:
Regarding this test, is it something that should be changed, or is it just a change that was added to this commit by mistake?

Reply:
Indeed, thanks